torch/_functorch/_aot_autograd/autograd_cache.py

```diff
@@ -151,6 +157,10 @@ def is_torch_function(target):
             return True
         if is_public_torch_api(target):
             return True
+        # Technically, FXGraphCache._check_for_hop already checks this,
+        # but better to error earlier anyway
```

Comment:
just curious - what extra work happens between here and the eventual call to `FXGraphCache._check_for_hop`?

Reply:
Not much, just going through the entire dynamo graph and checking for other ops.